execution_workflows/APA-Scan/configuration.ini

```diff
@@ -0,0 +1,29 @@
+[INPUT_RNAseq]
+# Input folder names
+# All samples(names like sample1_1.bam, sample1_2.bam....) in group1 must be inside of one folder, put the diectory on input1 parameter
+# All samples(names like sample1_1.bam, sample2_2.bam....) in group2 must be inside of one folder, put the diectory on input2 parameter
+input1 = /home/bs14e3m/projects/other/APAeval/execution_workflows/APA-Scan/srsf3

```

Comment:
Not good to put absolute file paths of your own system in a Git repo

Reply:
So I should have put the explaination in the file itself, the configuration.ini is run-specific, and is created by editing the blank_configuration.ini file in a rule in the Snakefile. The snakemake script is written to replace a placeholder (for input1 its input_dir1, with the current working dir of the pipeline. Wasn't 100% sure if this would work on the cloud but can just relplace the os.getcwd() with whatever dir you want. APA-Scan.py also wouldnt seem to work without specifying an absolute path hence the need for it here.